spec/models/aroc_week_4_spec.rb

```diff
@@ -26,14 +27,15 @@
 
   it '22. calculates the total sales for all but one user' do
     # ---------------------- Using Ruby -------------------------
-    orders = Order.all.map do |order|
-      order if order.user_id != @user_2.id
-    end.select{|i| !i.nil?}
-    total_sales = orders.map(&:amount).inject(:+)
+    # orders = Order.all.map do |order|
+    #   order if order.user_id != @user_2.id
+    # end.select{|i| !i.nil?}
+    # total_sales = orders.map(&:amount).inject(:+)
     # -----------------------------------------------------------
 
     # ------------------ Using ActiveRecord ---------------------
     # Solution goes here
+    total_sales = Order.where(user_id: [@user_1.id, @user_3.id]).sum(:amount).to_i
```

Comment:
You don't want to list out all the users you want because this isn't scalable. If you have a million users, you don't want to list out 999,999,999 user ids. Instead just list the one you don't want: `Order.where.not(user: @user_2)`